### ISSUE TITLE
Improve the contributor terms page with a summary

### ISSUE THREAD

**Issue by naoliv** [openstreetmap/openstreetmap-website#1107]
We were trying to understand some problems here in Brazil and noticed that people are basically accepting the [contributor terms](https://www.openstreetmap.org/user/terms) without reading.

It's possible to see that the [human readable summary](http://wiki.osmfoundation.org/wiki/License/Contributor_Terms_Summary) doesn't have a proper highlight in the page:

![Summary](http://i.imgur.com/8HlNzKf.png)
I really don't think that new users will click at this link in the footer.

One possible improvement would be to first display the summary, before the full contributor terms.

It could be as Creative Commons does for their licenses, like http://creativecommons.org/licenses/by/4.0/, using really simple words and as concise as possible.

I see two good points in doing this:
* the user could quickly have an idea of what he can and cannot do, even if they keep accepting without reading the full terms
* it should allow faster translations for the summary (since it isn't a technical language nor will have many words)

Could we have this kind of improvement, please?

**Comment by tomhughes** [comment]
Mostly this is something you would have to take up with the Licensing Working Group because we wouldn't be able to make any changes here without their prior approval.

The problem is that the formal terms are what people are agreeing to, so it's kind of important for people to understand that and not to agree based on a third party summary that we don't even have any control over.

**Comment by naoliv** [comment]
Of course I didn't mean to have people agreeing on the summary only, but I was thinking a page having something like this:
```
In summary you can:
* A
* B
* C

You can't:
* X
* Y
* Z

Full text goes here

[Agree] [Disagree]
```

I will contact the LWG.

**Comment by simonpoole** [comment]
Maybe before dicussing a solution it would be better to know what exactly the behavioural problem is? 

Note: the translations are limited in this case for legal reasons (the minimum set required). 

Note: signed up users are shown https://www.openstreetmap.org/welcome typically in their local language, this would likely be a better place for non-legal text.

**Comment by naoliv** [comment]
We are seeing people who are unaware they can't copy data from other places.
The terms are in English (some people can't read), long and technical. They just accept it.

We thought that having a simple summary¹ at the same page would help people understanding what they could and could not do in OSM.

We could translate this summary to other languages and still keep the original terms in English.

¹ Like CC does: simple phrases, explicitly saying that the summary doesn't substitute the license

**Comment by simonpoole** [comment]
@naoliv the welcome page already says that they can only copy from legit sources. Not that I'm saying no to your request, it is just not clear to me why you believe that the people in question will care about any text we put in the signup process.

**Comment by simonpoole** [comment]
@naoliv I've added the topic to the LWG todo list and have asked for input from the other members. As said as a short term measure I would suggest looking at your translation of the welcome page maybe simply highlighting the relevant text would help, further you could potentially put stronger language in the registration in e-mail about reading the welcome page.

Pro memoria, the current English text on the welcome page for stuff that shouldn't be added to OSM is:

    What it doesn't include is opinionated data like ratings, historical or hypothetical 
    features, and data from copyrighted sources. Unless you have special permission, 
    don't copy from online or paper maps. 

Maybe that could be made clearer for example (in English)

    What it doesn't include is opinionated data like ratings, historical or hypothetical 
    features, and data from copyrighted and other protected sources, for example 
    google maps. Unless you have special permission, don't copy from any online 
    or paper maps. 

Not that I'm a particular fan of mentioning a competitor on our website.

**Comment by naoliv** [comment]
Right, I will discuss with other mappers and see if we can improve our translation.
